webapp/src/translations/languages/en.json

```diff
@@ -782,7 +782,7 @@
         "pending": "Pending",
         "updateAvailableLabel": "Update available",
         "includeRewards": "Include reward transactions",
-        "loading": "Loading...",
+        "loading": "Loading ...",
```

Comment:
"Loading..." should remain as "Loading...", no `space`.

Reply:
You are right. I did some research and removed the space before the ellipsis.